hack/actions/install-kubernetes-toolchain.sh

```diff
@@ -56,6 +56,9 @@ if [[ ${GITHUB_ACTIONS} == "true" && ${OS} == "linux" ]]; then
   mv /home/runner/go/bin/ginkgo ${DESTDIR}/ginkgo
 fi
 
+# Install mockery for generating go mocks
+go install github.com/vektra/mockery/v2@v2.9.4
+
```

Comment:
I believe this can be moved into `tools.go` and managed as a normal go module dependency